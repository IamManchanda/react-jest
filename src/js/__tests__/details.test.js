import React from 'react';
import { create } from 'react-test-renderer';
import Details from '../pages/details';

test('snapshot', () => {
  const wrapper = create(<Details />);
  expect(wrapper.toJSON()).toMatchSnapshot();
});

test('show modal when toggalModal is called', () => {
  const wrapper = create(<Details search={() => {}} />);
  const instance = wrapper.getInstance();

  expect(instance.state.showModal).toBe(false);
  instance.toggleModal();
  expect(instance.state.showModal).toBe(true);
});
