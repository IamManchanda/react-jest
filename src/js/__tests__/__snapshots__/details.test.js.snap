// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<h1
  className="loading"
>
  Loading...
</h1>
`;
